src/theme/index.ts

```diff
@@ -1,2 +1,20 @@
-export { default as motoScout24Theme } from './ms24';
-export { default as autoScout24Theme } from './as24';
+import { theme as chakraDefaultTheme, extendTheme } from '@chakra-ui/react';
+
+import motoScout24Theme from './ms24';
+import autoScout24Theme from './as24';
+
+const chakraTheme = extendTheme({ name: 'Chakra' }, chakraDefaultTheme);
+const motoScoutChakraTheme = extendTheme(
```

Comment:
does `extendTheme` use the chakraDefaultTheme implicitly?

Reply:
that's the way I understood it, it extends the chakra theme with the passed overrrides